src/CSS/Header.css

```diff
@@ -3,4 +3,18 @@ header {
   font-family: 'Oswald', sans-serif;
   display: flex;
   justify-content: space-between;
-}
\ No newline at end of file
+}
+
+.tomato {
```

Comment:
We shoul try and look into how we can have the circle border so that only the tomato shows up. Probably after the major components have been dealt with first though.